docs/resources/er_instance.md

```diff
@@ -0,0 +1,92 @@
+---
+subcategory: "Enterprise Router (ER)"
+---
+
+# huaweicloud_er_instance
+
+Manages an enterprise router instance resource within HuaweiCloud.
```

Comment:
"enterprise router" is the service name and should be capitalize the first letter.

Reply:
done.